packages/paste-core/components/button/src/index.tsx

```diff
@@ -112,7 +112,13 @@ const handlePropValidation = ({
   }
 };
 
+const variantsWithoutBoundingBox = new Set(['reset', 'link', 'destructive_link']);
+
 const ButtonContents: React.FC<ButtonContentsProps> = ({buttonState, children, showLoading, variant}) => {
+  const buttonVariantHasBoundingBox = React.useMemo(() => variant && variantsWithoutBoundingBox.has(variant), [
```

Comment:
The Set `has()` function will return false for `undefined` and `null`, and this isn't very costly so we can avoid memoizing.

Simply doing `variantsWithoutBoundingBox.has(variant)` inline on line 130 should be enough

Reply:
Hmm so I made this change but TS is unsatisfied since `variant` can be `undefined`. Do you mind if I leave as is?

Reply:
TS > me.  But we can remove the `useMemo` still